terraform/modules/provider.tf

```diff
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = "~> 5.0"
+      version = "~> 6.0"
```

Comment:
This one looks like a major version change, do we expect any breaking changes? do we plan to test in `Dev` first?